Gordon360/Models/CCT/Minors.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Keyless]
+    public partial class Minors
+    {
+        [StringLength(5)]
+        [Unicode(false)]
+        public string MinorCode { get; set; }
+        [StringLength(50)]
+        [Unicode(false)]
+        public string MinorDescription { get; set; }
+    }
+}
```

Comment:
Why are you creating this one? Did you add this view to the database? Also, shouldn't this one have the `[Table("Minors", Schema = "dbo")]` key as well if you are adding it?

Reply:
Yes, but my real question is...why is it appearing in your pull request. I would pull it out since it is not something you are using.